test/it/DicomMessage.readFile.test.js

```diff
@@ -0,0 +1,39 @@
+import fs from 'fs';
+import path from 'path';
+import dcmjs from "../../src/index.js";
+
+const { DicomMessage } = dcmjs.data;
+
+describe('test parsing of sample-dicom.dcm file', () => {
+    const dicomTestFilesDataPath = path.join(__dirname, './../sample-dicom.dcm');
+
+    const arrayBuffer = fs.readFileSync(dicomTestFilesDataPath).buffer;
+    const dicomDict = DicomMessage.readFile(arrayBuffer);
+
+    it('has dict and meta section', () => {
+        expect(dicomDict['dict']).not.toBeNull();
+        expect(dicomDict['meta']).not.toBeNull();
+    })
+
+    describe('test some tags from the dict section', () => {
+        const dictTags = dicomDict['dict'];
+
+        it('has the StudyDate', () => {
+            expect(dictTags).toHaveProperty('00080020.Value[0]', '20010101');
+        })
+        // testing nested sequence parsing
+        it('has the  RequestAttributesSequence -> ScheduledProtocolCodeSequence -> CodeValue', () => {
+            expect(dictTags).toHaveProperty('00400275.Value[0].00400008.Value[0].00080100.Value[0]', '6310');
+        })
+
+    })
+
+    describe('test some tags from the meta section', () => {
+        const metaTags = dicomDict['meta'];
+
+        it('has the TransferSyntaxUID', () => {
+            expect(metaTags).toHaveProperty('00020010.Value[0]', '1.2.840.10008.1.2');
+        })
+
+    })
+})
```

Comment:
Thanks so much for the test 👍 

I like the idea of breaking up the tests a bit to keep things organized.  The folder name `it` isn't clear to me though.  I'd prefer calling it `integration` or `integrationTests`.

Reply:
I think the naming is really up to the team that organizes the code. At least, I did not found a common design pattern yet. 

The "it" was used in some articles, like here: [Split unit and integration tests](https://blog.worldline.tech/2020/04/10/split-unit-and-integration-tests.html). 

I like "integration" as well. I changed the folder from "it" to "integration".

Reply:
Yes, this is great.